coderd/rbac/authz_test.go

```diff
@@ -570,29 +638,56 @@ func testAuthorize(t *testing.T, name string, subject subject, sets ...[]authTes
 		for _, c := range cases {
 			t.Run(name, func(t *testing.T) {
 				for _, a := range c.actions {
-					err := authorizer.Authorize(context.Background(), subject.UserID, subject.Roles, a, c.resource)
-					if c.allow {
-						if err != nil {
-							var uerr *rbac.UnauthorizedError
-							xerrors.As(err, &uerr)
-							d, _ := json.Marshal(uerr.Input())
-							t.Logf("input: %s", string(d))
-							t.Logf("internal error: %+v", uerr.Internal().Error())
-							t.Logf("output: %+v", uerr.Output())
-						}
-						require.NoError(t, err, "expected no error for testcase action %s", a)
-						continue
-					}
-
-					if err == nil {
+					ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitShort)
+					t.Cleanup(cancel)
+					authError := authorizer.Authorize(ctx, subject.UserID, subject.Roles, a, c.resource)
+					// Logging only
+					if authError != nil {
+						var uerr *rbac.UnauthorizedError
+						xerrors.As(authError, &uerr)
+						d, _ := json.Marshal(uerr.Input())
+						t.Logf("input: %s", string(d))
+						t.Logf("internal error: %+v", uerr.Internal().Error())
+						t.Logf("output: %+v", uerr.Output())
+					} else {
 						d, _ := json.Marshal(map[string]interface{}{
 							"subject": subject,
 							"object":  c.resource,
 							"action":  a,
 						})
 						t.Log(string(d))
 					}
-					require.Error(t, err, "expected unauthorized")
+
+					if c.allow {
+						require.NoError(t, authError, "expected no error for testcase action %s", a)
```

Comment:
should these be `assert.NoError` instead?  I don't see how they break the rest of the test or other test cases